### ISSUE TITLE
Local minima: white image output

### ISSUE THREAD

**Issue by BartvanderWoude** [BartvanderWoude/Complexity-pretext-task-on-organoid-segmentation#18]
Models tend to quickly get stuck producing white images as output. Sometimes they are able to break through this minimum and train properly. Yet unsure why this happens and how it can be solved?

**Comment by BartvanderWoude** [comment]
Adding batch normalization and possibly dropout to hopefully solve this issue.
